wlauto/external/uiauto/src/com/arm/wlauto/uiauto/BaseUiAutomation.java

```diff
@@ -37,6 +38,28 @@
 
 public class BaseUiAutomation extends UiAutomatorTestCase {   
 
+    public class ActionLogger {
```

Comment:
Maybe add a docstring  with some documentation describing what it is good for, and how it is intended to be used.